src/components/LoginRedirect.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState, useContext } from 'react'
+import { navigate, useLocation, useParams } from '@reach/router'
+import Context from '../modules/Context'
+const { apiURL } = require('../config.json')
+
+const LoginRedirect = (props) => {
+  const [text, setText] = useState('Loading...')
+  const location = useLocation()
+  const params = useParams()
+  const { dispatch } = useContext(Context)
+
+  useEffect(() => {
+    // Successfully logged with the provider
+    // Now logging with strapi by using the access_token (given by the provider) in props.location.search
+    fetch(`${apiURL}/auth/okta/callback${location.search}`)
```

Comment:
Axios has the ability to intercept HTTP requests. Fetch, by default, doesn't provide a way to intercept requests. It would be better if we use axios here.